infra/gcp/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,56 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
do we need this file? if so, why was it not there before?

Reply:
it was not needed before. We changed it when we provide k8s support.